consent-ui/src/app/errorsHandler/global-errors-handler.ts

```diff
@@ -9,11 +9,11 @@ export class GlobalErrorHandler implements ErrorHandler {
 
   handleError(error) {
     console.error(error);
+    let message = 'Something went wrong';
+
     const errorService = this.injector.get(ErrorService);
     const infoService = this.injector.get(InfoService);
 
-    let message = 'Something went wrong';
-
     if (error instanceof HttpErrorResponse) {
```

Comment:
It is better to make check here` if (error !== null){...}` such that we have `Something went wrong` as default message